src/Microsoft.IdentityModel.JsonWebTokens/JwtConstants.cs

```diff
@@ -32,6 +32,11 @@ namespace Microsoft.IdentityModel.JsonWebTokens
     /// </summary>
     public static class JwtConstants
     {
+        /// <summary>
+        /// Token content type.
+        /// </summary>
+        public const string ContentType = "JWT";
```

Comment:
I don't think we need this, we can use: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/53e19a887bfed8ee55a91f747494cf419c10f97f/src/Microsoft.IdentityModel.JsonWebTokens/JwtConstants.cs#L38

Reply:
As it is type of the 'Content' and not type of the 'Header' I've created a new const( Following the same pattern of creating a new constant for TokenType as well 😃) But I see the argument of reusing the existing constant. I can go ahead change it if we think using HeaderType should be ok.

Reply:
I think we should stick with HeaderType.

Reply:
Since the default value for both the 'typ' and 'cty' claim is always going to be the same, I think using the same constant is okay. I think modifying the comment to mention this information would be useful though.

It also looks like the 'TokenType' constant isn't actually used anywhere.

Reply:
I went ahead removed ContentType constant.